src/openshift/component.ts

```diff
@@ -208,12 +209,23 @@ export class Component extends OpenShiftItem {
         if (await Component.odo.isPodmanPresent()) {
             return Component.devRunOn(component, 'podman');
         }
-        void window.showErrorMessage('Podman is not present in the system, please install podman on your machine and try again.', 'Install podman')
-            .then(async (result) => {
-                if (result === 'Install podman') {
-                    await commands.executeCommand('vscode.open', Uri.parse('https://podman.io/'));
-                }
-            });
+        const podmanOnPath = which('podman');
+        if (podmanOnPath) {
+            const SETUP_INSTRUCTIONS = 'Open setup instructions';
+            void window.showErrorMessage('Podman is present on the system, but is not fully set up yet.', SETUP_INSTRUCTIONS)
```

Comment:
@jeffmaury what do you think about this wording?

Reply:
I don't think it's gonna fix the issue as `Component.odo.isPodmanPresent` would return `true` and code won't be executed

Reply:
`odo version` only seems to report podman as installed if the podman machine is present, so in the case that the binary is on the path but the machine is not set up, it will go to this case. I've tested it on my Windows machine.

Reply:
@jeffmaury if you have time, could you please tell what you think of the wording of this message? I'll double check that it shows up under the right circumstances.